package.json

```diff
@@ -221,6 +221,18 @@
       ]
     },
     "commands": [
+      {
+        "command": "tabnine.chat.submit-message",
+        "title": "Submit message"
+      },
+      {
+        "command": "tabnine.chat.state",
+        "title": "Chat global state"
+      },
+      {
+        "command": "tabnine.chat.clear-all-conversations",
+        "title": "Clear all tabnine chat conversations"
+      },
```

Comment:
Isn't this available to the users in the search bar as well?
maybe add a condition in the "when" field, and set it to true only when the eval is running?

Reply:
agree with @yonip23, another thing - the vscod's practice of exposing apis, is to return an object from the activate function, see an example usage here https://github.com/codota/tabnine-vscode/blob/b4e19e26e2cbd85bfa16aff075975e00d1b13df5/src/languages/java/JavaHome.ts#L14

Reply:
im not sure i undestand what is the change you want here @dimacodota

Reply:
@dimacodota Chose to avoid refactoring all the code to expose these methods from the activate function, Used when...